Sources/NoCodes/Delegate.swift

```diff
@@ -40,6 +40,10 @@ extension NoCodes {
     /// Called when NoCodes flow is finished and the NoCodes screen is closed
     func noCodesFinished()
     
+    /// Called when NoCodes screen loading failed
+    /// Don't forget to close the screen. `NoCodes.shared.close()`
```

Comment:
```suggestion
    /// Don't forget to close the screen using `NoCodes.shared.close()`
```

Reply:
Also, if we're sure that the user needs to close the screen, why don't we close it ourselves?

Reply:
I thinks it's better to let this option for our users. So they can show alert and then close screen, for example.